app/components/post/index.js

```diff
@@ -84,6 +84,7 @@ function makeMapStateToProps() {
             isFlagged: isPostFlagged(post.id, myPreferences),
             isCommentMention,
             isLandscape: isLandscape(state),
+            previousPostExists: (previousPost ? true : false),
```

Comment:
You can simplify this a bit by changing it to
```suggestion
            previousPostExists: Boolean(previousPost),
```

Reply:
Thanks for the suggestion @hmhealey, added this!